lib/storage/storage.go

```diff
@@ -94,3 +94,8 @@ func ParseConfig(s string) (u *Config, err error) {
 
 	return
 }
+
+type CommittableBackend interface {
```

Comment:
What is the `Backend` mean?

Reply:
@Charleslee522 Oh, it should be fixed to 'CommittableCore', this means, `LevelDBCore` , which has `Commit` and `Discard`.

Reply:
Why not just `Commitable` ?

Reply:
@Geod24 Ooops. My poor english :)

Reply:
Oops, this was not about `Commitable` vs `Committable` (the former is a typo, as @Charleslee522 realized!), but `CommittableCore` vs `Committable`.